.config/webpack/utils.ts

```diff
@@ -0,0 +1,40 @@
+import fs from 'fs';
+import path from 'path';
+import util from 'util';
+import { glob } from 'glob';
+import { SOURCE_DIR } from './constants';
+
+export function getPackageJson() {
+  return require(path.resolve(process.cwd(), 'package.json'));
+}
+
+export function getPluginJson() {
+  return require(path.resolve(process.cwd(), `${SOURCE_DIR}/plugin.json`));
+}
+
+export function hasReadme() {
+  return fs.existsSync(path.resolve(process.cwd(), SOURCE_DIR, 'README.md'));
+}
+
+// Support bundling nested plugins by finding all plugin.json files in src directory
+// then checking for a sibling module.[jt]sx? file.
+export async function getEntries(): Promise<Record<string, string>> {
+  const pluginsJson = await glob('**/src/**/plugin.json', { absolute: true });
```

Comment:
This is weird... You have a utility function defined on 11 to find the file, then you're globbing for the file?